lib/cli-engine/config-array-factory.js

```diff
@@ -894,11 +894,10 @@ class ConfigArrayFactory {
      * @param {ConfigArrayFactoryLoadingContext} ctx The loading context.
      * @returns {DependentParser} The loaded parser.
      */
-    _loadParser(nameOrPath, ctx) {
+    static loadParser(nameOrPath, ctx) {
```

Comment:
I made the method static because the linter complained about no usage of this in the method body.